shared/components/Speaker/index.js

```diff
@@ -1,40 +1,33 @@
 import React, { PropTypes } from 'react';
 
-const IconLink = ({ present, url, type }) => {
-  if (!present) { return null; }
+const IconLink = ({ url, type }) => {
+  if (!url) { return null; }
```

Comment:
This won't work for the github and twitter one as the URL is always present, though sometimes not correct.